h/celery.py

```diff
@@ -98,12 +98,6 @@ def bootstrap_worker(sender, **kwargs):
 
 
 @signals.task_prerun.connect
-def reset_feature_flags(sender, **kwargs):
```

Comment:
Just to check. We don't need this anymore because the LRU cache clears automatically when a transaction ends and celery tasks run in their own transaction?